### ISSUE TITLE
Printed Campaign Guides?

### ISSUE THREAD

**Issue by brightbold** [Inexorable-Fate/mpc_projects#145]
### Is your feature request related to a problem? Please describe.

People always ask about campaign guides. You can print the campaign or investigator expansion, but we don't provide a way to get nice versions of the guides.

### Describe the solution you’d like

MPC only accommodates A4 size (most guides are square) and max 24 pages, which is way to short for an Arkham campaign. But is there a way we could link people elsewhere to print? CYOA guides print at Lulu & B&N, so possibly we could format for one of these. In addition we might be able to link to CYOA guides as well, provided we have some kind of "for personal use only" disclaimer.

### Describe alternatives you’ve considered

See MPC, above.

### Additional context

One flaw is that printing the guides requires them to be in multiples of 4 pages (for saddle stitched) or 2 (for spiral bound), and actual guides typically just take as many pages as they need, often even an odd number. So we'd need a plan for that.

**Comment by Bonzman72** [comment]
Could we add a backpage for all odd numbered campaign guides. Just like the classic starry background, nothing complex. Maybe add a blurb onto it or the encounter icons. Would at least make the spiral bound guides possible to print.

**Comment by Bonzman72** [comment]
Or we can create front covers like the official campaigns?

**Comment by Bonzman72** [comment]
Something like this (just a mock up in photoshop)<img width="612" alt="Image" src="https://github.com/user-attachments/assets/b9e70488-baca-4256-99b1-346b67193533" />

**Comment by Bonzman72** [comment]
Some examples:
<img width="612" alt="Image" src="https://github.com/user-attachments/assets/310dca63-e3c7-4655-9f9d-d182f6d35f8e" />
<img width="612" alt="Image" src="https://github.com/user-attachments/assets/92818904-f538-4c61-9118-02cdc0e5e478" />

**Comment by Bonzman72** [comment]
Just thinking, to print most campaigns the guides will have to be rewritten as A4 instead of square. I looked into lulu as well which could work for spiral bound guides.

**Comment by brightbold** [comment]
Great ideas on how to add extra pages if needed! We could need up to 3 for saddle-stitch or only 1 for spiral bound. Another thought I had was that sometimes people don't do campaign logs and we could add a fairly generic one (maybe using the Dunwich format, which doesn't have any extra elements or art) — I have that font so it would be easy to customize a campaign log with a little erasing.

The rewriting as A4 would require us to get the SE files from the authors and do a lot of reworking, so I was thinking maybe we'd wait on this step until the highres templates are implemented, at which point I'd love to re-export at least all the popular campaigns.

My memory is Lulu had a format that was 1/2" off on one side from the square ones so I think we could do that without reformatting — just adding some extra margin art? Does that track with your research?

**Comment by Bonzman72** [comment]
That's a good idea, campaign logs are normally missing. Also, for a final page, normally the encounter sets on the back are missing too.

That's a good point about rewriting, it would be a lot of work.

I didn't actually look at square guides, I was trying to get them as close to the official as possible but I can look. One thing we could do is put out a quiet message on the MB discord just asking campaign creators to use the A4 format?

**Comment by brightbold** [comment]
Haha I came here to add my idea of adding the list of encounter sets that people always leave off but you already thought of that!

**Comment by Bonzman72** [comment]
@brightbold While waiting for higher res templates, we could always get the campaigns prepared for reformatting/rewriting. We could get all the text written on word documents if need be so it could just be copy and pasted, locate the relevant artwork, make the front covers and campaign logs, and locate all the encounter icons. It would be a lot of work but it would pay off.

Also, for standalone campaigns, it might be worth collecting standalone campaigns into groups that could be printed like the campaign guides. For example "Standalone scenarios Vol 1".

**Comment by Bonzman72** [comment]
(Only rewrite the text if we cannot get the eons files)

**Comment by brightbold** [comment]
I have a fantasy that once the highres templates are in SE (Soon™)* we will ~~insist upon~~ offer to re-export all the popular campaigns with the new templates (and Arno Pro!) and that that would be a good opportunity to make any campaign guide updates that are needed. I thought about mentioning that in that conversation but I felt like Dr. Jack already felt piled on and I decided later would be better. 

But I think the spin in his case is that there's a difference between tweaking forever and improving setup — we would not be rebalancing for difficulty or any of the other things he was talking about, just taking feedback on where the instructions could be made clearer. (There's also one set of cards that should be a different encounter set — I can't remember if it was merging two sets, separating two, or changing a name but I remember there was an issue — that was where I got the "no I'm not making more changes" info from him but we might be able to squeak that change in too as it's related to setup and not mechanics. 

I understand his overall point of not wanting people to feel they need to reprint but high res templates are going to make me want to reprint everything so they're already introducing that problem.


\* _Note: not actually soon because I had the dumb idea to run Arkham Con ’25 instead of doing the templates project_

**Comment by brightbold** [comment]
> [@brightbold](https://github.com/brightbold) While waiting for higher res templates, we could always get the campaigns prepared for reformatting/rewriting. We could get all the text written on word documents if need be so it could just be copy and pasted, locate the relevant artwork, make the front covers and campaign logs, and locate all the encounter icons. It would be a lot of work but it would pay off.

Yeah this is a great idea.

Also Tokeeto has made a campaign guide template in Open Office as a way to improve the PDF issue (Strange Eons makes an image of each campaign guide page and then creates a PDF of images, so none of the text is selectable or searchable.) So if you were going to be retyping campaign guides anyway you could just do it in his template and then no need to reformat in SE which is a pain. (Although I've put in a hanging indents feature request, so if that gets in it will be much less of a pain.)

**Comment by brightbold** [comment]
But... it's possible Tokeeto will do something better with campaign guides, per [this comment](https://discord.com/channels/225349059689447425/249270867522093056/1326609324147081316). So it could turn out that getting started early might end up necessitating some rework.

> Personally, I'm using LibreOffice right now, to create a campaign guide.
> It's not quite there yet, but for campaign guides I find the experience a lot more pleasant that SE, since it's a writing tool, not just a layout tool.
>
> But I might need to take a look at LaTeX, if I wanna do this right... 😮‍💨

**Comment by dancuadra** [comment]
I have access to Acrobat Pro. I tried OCR on the Jumanji campaign guide as
an example. It works, but when you copy the text to a blank document, there
is a lot of cleanup necessary. Would require careful proofreading, but
still less work than rewriting everything from scratch.

**Comment by Bonzman72** [comment]
Yeah, I tried writing out a standalone just to see if it could be done. It took about an hour and a half so doable but painstaking. If we copy and paste the big chunks, maybe that'll leave less formatting errors than copy and pasting the whole thing?

**Comment by dancuadra** [comment]
Strangely, when I run the OCR the text looks fine in the PDF, but if I copy-paste it to a doc it introduces misspellings that would have to be corrected. Same results regardless of how much text you copy at a time.
![Image](https://github.com/user-attachments/assets/e7c0353c-ce81-4ad3-ac41-a977aff9cc81)

**Comment by Bonzman72** [comment]
hmm, thats odd. I used the online version and didnt see an issue with copy and pasting. Is it worth checking another campaign guide?

**Comment by Bonzman72** [comment]
Just a thought, could we get some of the campaign and stand alone text from Zzorba? The arkham cards github should have the info right?
